src/utils/themeFileLocation.js

```diff
@@ -0,0 +1,8 @@
+export const getFiles = () =>
+  require.context(
+    '/Users/dandrory/Work/development-environment/frontend_dir/branding',
```

Comment:
This meant to be here?

Reply:
nope it is not :D. Should be replaced with the define plugin stuff!